sdk/typescript/src/builder/Inputs.ts

```diff
@@ -2,8 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { array, boolean, Infer, integer, object, string, union } from 'superstruct';
-import { normalizeSuiAddress, ObjectId, SharedObjectRef, SuiObjectRef } from '../types';
-import { builder } from './bcs';
+import { normalizeSuiAddress, ObjectId, SharedObjectRef, SuiObjectRef } from '../types/index.js';
```

Comment:
I assume this is for actual ESM compatibility? I'm fine with this (bummed but I don't think there's an alternative). Is this linted against?

Reply:
yup, for eslint compatibility (didn't need it before because we bundled to a single file).  I added an eslint rule to enforce this